5calls/app/src/main/java/org/a5calls/android/a5calls/controller/IssueActivity.java

```diff
@@ -251,7 +253,8 @@ public void onBackPressed() {
     private void reportCall(String callType, String zip) {
         setButtonsEnabled(false);
         AppSingleton.getInstance(getApplicationContext()).getDatabaseHelper().addCall(mIssue.id,
-                mIssue.contacts[mActiveContactIndex].id, callType, zip);
+                mIssue.name, mIssue.contacts[mActiveContactIndex].id,
```

Comment:
Kind of odd to have commas mid-line.  I think the arguments should either be all on the same line or all on their own line.